code/modules/gear_presets/uscm_medical.dm

```diff
@@ -91,15 +77,10 @@
 	if (new_human.client && new_human.client.prefs && (new_human.client.prefs.backbag == 1))
 		back_item = /obj/item/storage/backpack/marine
 
-	new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/almayer/doc(new_human), WEAR_L_EAR)
 	new_human.equip_to_slot_or_del(new back_item(new_human), WEAR_BACK)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/rank/medical/green(new_human), WEAR_BODY)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/white(new_human), WEAR_FEET)
 
-	if(new_human.disabilities & NEARSIGHTED)
-		new_human.equip_to_slot_or_del(new /obj/item/clothing/glasses/hud/health/prescription(new_human), WEAR_EYES)
-	else
-		new_human.equip_to_slot_or_del(new /obj/item/clothing/glasses/hud/health(new_human), WEAR_EYES)
```

Comment:
Why remove this? At least give them prescription glasses if they have the trait on spawn. Or implement their vendors to give them the prescription variant. Or allow buying a `/obj/item/clothing/glasses/regular` (this and `/obj/item/clothing/glasses/sunglasses/prescription` can be inserted into any `/obj/item/clothing/glasses` to give it prescription element.